chore: link to wallet docs, update graphics (#1191)
WalletConnect/walletconnect-docs: docs/web3wallet/notify/installation.mdx

Web3Inbox is in beta
:::

> The WalletConnect Notify API allows wallet developers to build engaging notification experiences from dapps directly to wallet addresses giving the end user the ability to subscribe to and receive notifications in their wallet.

Reviewer: Let's shorten this to be more direct.

> The WalletConnect Notify API allows wallet developers to build engaging notification experiences from dapps directly to wallet addresses giving the end user the ability to subscribe to and receive notifications in their wallet.

Reviewer: ```suggestion
The WalletConnect Notify API allows wallet developers to build engaging notification experiences with dapps. It enables the end users to subscribe to and receive notifications to their wallet.
```

> The SDK is cross-platform supporting iOS and Android wallets.

Reviewer: ```suggestion
For those looking to add notifications for session requests and session updates, please check [here](../wallet-usage#extra-platform-specific). 

The SDK is cross-platform supporting iOS and Android wallets.
```

Author: Do they know what a "session request" and a "session update" are? What about:

```
For those looking to add notifications for pairing and sign requests
```
